src/db/dbQuery.ts

```diff
@@ -0,0 +1,39 @@
+import { database } from "./dbConfig";
+
+
+export const getAllUnsentEmailData =(email:boolean = true, email_sent:boolean = false, seen:boolean = false) => {
```

Comment:
```suggestion
export const getAllUnsentEmailData =() => {
```
I don't think we want to parametize that. So it is better to hardcode them.